contentpacks/utils.py

```diff
@@ -78,6 +83,38 @@ def compute_translated(self, pofile: polib._BaseFile) -> int:
 
         return (trans_count / all_strings_count) * 100
 
+    def compute_interface_translated_percentage(self, lang, version):
+        """
+        Fetch the translated status from CrowdIn API then calculate the percentage of the translated words from
+        the {version}-djangojs.po and {version}-django.po.
+        Returns the calculated percentage.
+        """
+        crowdin_url = CROWDIN_API.format(crowdin_project=KALITE_PROJECT, crowdin_secret_key=KALITE_SECRET_KEY,
+                                         lang=lang)
+        logging.info("Fetch crowdin status from (%s)" % crowdin_url)
+        crowdin_data = urllib.request.urlopen(crowdin_url)
+        data = ujson.loads(crowdin_data.read())
+        version_files = data["files"][0]
+        total_words_approved = 0
+        total_words = 0
+        djangojs_po = "%s-djangojs.po" % version
+        django_po = "%s-django.po" % version
+        # Loop on the `files` dict to get the translated status.
+        for filenames in version_files.get("files"):
```

Comment:
I think the `filenames` here should be in singular form - I'm actually confused if it's a dictionary or list of filenames.